.eslintrc.js

```diff
@@ -15,7 +15,15 @@ module.exports = {
         'plugin:@angular-eslint/template/process-inline-templates',
         'plugin:@isyfact/recommended',
         'plugin:editorconfig/all'
-      ]
+      ],
+      rules: {
+        '@typescript-eslint/unbound-method': [
+          'error',
+          {
+            ignoreStatic: true
+          }
+        ]
+      }
```

Comment:
If this is really the best possibility (doubt), then this would need to be merged into isy-eslint-plugin and removed afterwards imo

Reply:
This rule is used for an ESLint error inside the **productive** code of the demo application.

If we put this rule inside the isy-eslint-plugin every application must use this rule as recommended productive code rule.

That's wrong.

This rule is only used inside our demo application and must be declared inside the central isy-angular-widgets .eslintrc

Reply:
I don't get it. Either the rules makes sense or it does not - doesn't depend on prod code or not?

Reply:
It's a specific case inside the demo code.
It's not for global usage.
So this change has no place inside the isy-eslint repo.

Reply:
Why is this a rule specific for a demo case? What sets our demo case so much apart from standard code that it needs a rule customisation that does not make sense in a broader context?

Reply:
Even more concrete: In my understanding, every project using our validators will have the exact same problem.

Reply:
> Even more concrete: In my understanding, every project using our validators will have the exact same problem.

Yes that's right, since 2021 to disable the rule seems to be the best solution if using Angulars static Validators.*
Since we do a lot with forms in the government context, we can also include this in the global rules. 👍

Reply:
Ok, the rule is going to be included in the global rules.
*Rules must be merged and released first OR the rule must be deactivated inside the code